beacon-chain/rpc/prysm/v1alpha1/validator/unblinder.go

```diff
@@ -75,6 +79,9 @@ func (u *unblinder) unblindBuilderBlock(ctx context.Context) (interfaces.SignedB
 		return nil, errors.Wrap(err, "could not set sync aggregate")
 	}
 	sb.SetSignature(sig[:])
+	if err = sb.SetBLSToExecutionChanges(blsToExecChanges); err != nil && !errors.Is(err, consensusblocks.ErrUnsupportedGetter) {
```

Comment:
Wouldn't the error be unsupported Setter here instead of Getter?

Reply:
This requires some more refactoring, but I can add it.